### DISCUSSION TITLE
Why do 1.0 base stations require synchronization whereas 2.0 base stations don’t?

### DISCUSSION THREAD

**Discussion by apark2459** [bitcraze/discussions#1527] [General]
I’m working on interfacing base station 2.0 with custom hardware for robotics and was curious.

**Comment by ataffanel** [comment]
The reason is that the basestation V1 work by synchronizing 2 basestations and make them send sweeps one after each other (it is a kind of TDMA, Time Division Multiple Access). Each timeslots contains a synchronization pulse of both basestation and then one of the 4 sweep. This requires both basestation to be synchronized on the same timeslots and limits the number of basestations to 2 maximum.

Basestations V2 do not transmit a synchronization pulse, they transmit the synchronization information as a time offset in the laser beam modulation itself. This means that with V2 receiving a single pulse is enough to identify the basestation and the distance from the pulse to the synchronization instant of that basestation. So there is no need of synchronization anymore as long as basestations are setup on different channels. For added resilience, each channel runs at a slightly different rotation speed.

**Reply by apark2459** [reply]
Thank you for the clarification! Does that mean the base station has its own clock that "counts" the offset and encodes this offset in real time into the beams?

**Reply by ataffanel** [reply]
Yes, this is in essence what is happening. The base-station transmit a pseudorandom bit-stream generated by an LFSR at fixed bit-rate. By receiving enough bits of this LFSR we can know how far we are in the pseudorandom sequence and so how far in time we are from the sync instant.
